CONTRIBUTING.rst

```diff
@@ -53,24 +53,19 @@ Unless you are using Python 2.7, please also install the bugbear plugin:
 
 You can then run ``flake8`` directly as follows::
 
-    $ flake8 Bio/
-    $ flake8 BioSQL/
-    $ flake8 Tests/
-    $ flake8 Scripts/
-    $ flake8 Doc/examples/
-
-Each of these folders has its own flake8 settings file, but in the long term
-we would like all the code to follow the same strict settings.
+    $ flake8
 
 You can run the reStructuredText checks with the ``restructuredtext-lint``
```

Comment:
Maybe move this after the flake8 hook bit (pending a more sophisticated hook setup)?

Reply:
I would suggest to move everything from line 45-52 (installing flake8) directly below line 36, lines 64-68 (flake8 pre-commit hook) directly below that (with a minor rephrase), and then mention the flake 8 command line access. Finally the other stuff.
Like this: 
1. "We follow PEP-8 and PEP-257"
2. "You install flake8"
3. "You set up a pre-commit hook"
4. "You can also run flake8 independently"
5. "For docstrings, we are also using...."
....

Reply:
Sounds good - feel free to update this branch if I’ve not, but will try tomorrow